app/models/panel_condition.rb

```diff
@@ -0,0 +1,9 @@
+class PanelCondition < ApplicationRecord
+  before_save :panel_condition
```

Comment:
This would set all fields (tem_ext, temp_panel, radiation) to 0 before saving the record. Please check if this works as it should.